jhead: add page (#12001)
tldr-pages/tldr: pages/linux/jhead.md

# jhead

> Image timestamp and EXIF data manipulation.

> > More information: <https://www.sentex.net/~mwandel/jhead/>.

Reviewer: ```suggestion
> More information: <https://www.sentex.net/~mwandel/jhead/usage.html>.
```

Author: Done

> - Show all exif data:

Reviewer: ```suggestion
- Show all EXIF data:
```

Author: Done

`jhead {{path/to/image.jpg}}`

- Set the file's date and time to the EXIF create date (file creation date will be changed):

`jhead -ft {{path/to/image.jpg}}`

- Set the EXIF time to the file's date and time (EXIF data will be changed):

> `jhead -dsft {{path/to/image.jpg}}`

Reviewer 2: From the description, I think this is what you meant?

```suggestion
`jhead -dsft *.jpg`
```

> - Rename all jpeg files based on the exif create date to `YYYY_MM_DD-HH_MM_SS.jpg`:

Reviewer: ```suggestion
- Rename all JPEG files based on the EXIF create date to `YYYY_MM_DD-HH_MM_SS.jpg`:
```

Author: Done

> - Rename all jpeg files based on the EXIF create date to `YYYY_MM_DD-HH_MM_SS.jpg`:

Reviewer: ```suggestion
- Rename all JPEG files based on the EXIF create date to `YYYY_MM_DD-HH_MM_SS.jpg`:
```

`jhead -n%Y_%m_%d-%H_%M_%S *.jpg`

> - Lossless rotation of all jpeg images by 90, 180 or 270 based on the exif orientation tag:

Reviewer: ```suggestion
- Rotate losslessly all JPEP images by 90, 180 or 270 based on the EXIF orientation tag:
```

> - Rotate losslessly of all JPEG images by 90, 180 or 270 based on the EXIF orientation tag:

Reviewer: ```suggestion
- Rotate losslessly all JPEG images by 90, 180 or 270 based on the EXIF orientation tag:
```

`jhead -autorot *.jpg`

> - Update all exif timestamps (Format: +- hour:minute:seconds) (example: forgot to change the camera's time zone - removing 1 hour from timestamps):

Reviewer: ```suggestion
- Update all EXIF timestamps (Format: +- hour:minute:seconds) (example: forgot to change the camera's time zone - removing 1 hour from timestamps):
```

`jhead -ta-1:00:00 *.jpg`

> - Remove all exif data (including thumbnails):

Reviewer: ```suggestion
- Remove all EXIF data (including thumbnails):
```
